src/node_api.h

```diff
@@ -512,6 +512,10 @@ NAPI_EXTERN napi_status napi_cancel_async_work(napi_env env,
 // version management
 NAPI_EXTERN napi_status napi_get_version(napi_env env, uint32_t* result);
 
+// Garbage collection
```

Comment:
Needs more detail though.  Most people will wonder the same thing as @mhdawson when it's just one of many input signals that influence the frequency of GC runs.